tests/pipeline/nodes/model/yolov4/test_yolo.py

```diff
@@ -55,7 +55,7 @@ def yolo_detector(yolo_config):
 def replace_download_weights(root, blob_file):
     return False
 
-
+@pytest.mark.unit
```

Comment:
Should this be mark.mlmodel ?

Reply:
Yes, it was a mistake on my end. Thanks for pointing out